gems/pending/disk/modules/QcowDisk.rb

```diff
@@ -115,6 +115,9 @@ def d_init
     @dOffset = dInfo.offset
     @downstreamDisk = dInfo.downstreamDisk
     self.diskType = "#{diskType}-#{@downstreamDisk.diskType}" if @downstreamDisk
+
+    # save the handle for Qcow disks in case of opening their parent disks
```

Comment:
I would change this comment to something like:

# Ensure all the disks in the chain are opened before we return (required to address RVEV SSA UID issues).